desktop/containerd/index.md

```diff
@@ -209,6 +209,19 @@ View Tags on DockerHub to see multi-platform result:
 
 ## Containerd Image Store Release Notes
 
+(2022-10-19)  
+New commands and multiple bug fixes included with Docker Desktop 4.13.0.
+
+### New
+* Fixed: Kubernetes not starting.
+* Fixed: Kubernetes IN Docker (kind) not starting.
```

Comment:
Also, in keeping with our style guide, and other release notes, it'd be great if we could remove the ':' from all these bullet points and have complete sentences. For example, 'Fixed a bug which caused Kubernetes in Docker (kind) to not start'

Reply:
Well noted, done!